test/views/paye/PayeIncomeTaxAndNicsViewSpec.scala

```diff
@@ -81,5 +81,19 @@ class PayeIncomeTaxAndNicsViewSpec extends UnitSpec with OneAppPerSuite with Tes
       document.getElementById("totalUkIncomeTax").text() shouldBe "Total £20,322.00"
 
     }
+
+
+    "have correct data for national insurance contributions" in {
```

Comment:
What about if there is no NI contribution?

Reply:
agreed
done